hello-world-dotnet-task/readme.md

```diff
@@ -0,0 +1,27 @@
+# Hello World .NET
+
+Sample for Hello World task. Print "Hello World" when run the main.py
```

Comment:
lets update this

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/8c0bf8690a2860b6f806d4bcad197fe86f1792c7